views/dashboard-application.hbs

```diff
@@ -125,163 +123,135 @@
                                 <option {{#if user.yearOther}} selected{{/if}}>Other</option>
                             </select>
                         </div>
-                        <div class='form-group col-md-5'>
+                        <div class='form-group col-md-5 col-sm-10 col-xs-10'>
                             <label>GITHUB</label>
                             <input class='form-control' type='url' name='github' placeholder='GitHub' value='{{user.github}}'>
                         </div>
-
-                        <div class='form-group col-md-5'>
+                        <div class='form-group col-md-5 col-sm-10 col-xs-10'>
                             <label>LINKEDIN</label>
                             <input class='form-control' type='url' name='linkedin' placeholder='LinkedIn' value='{{user.linkedin}}'>
                         </div>
-                        <div class='form-group col-md-5'>
+                        <div class='form-group col-md-10 col-sm-10 col-xs-10'>
                             <label>RESUME</label>
                             <span class="row"></span>
-                            <a href="https://forms.gle/UDU6GJmQ5hQ9ykNN7" target="_blank"><button class='file-upload'>
-                                    UPLOAD FILE </button></a>
+                            <a href="https://forms.gle/UDU6GJmQ5hQ9ykNN7" target="_blank" style="text-decoration: none;"><button class='file-upload'> UPLOAD FILE </button></a>
                         </div>
                     </fieldset>
                 </div>
 
                 <!-- TEAM INFORMATION -->
-                <div class='row'>
-                    <div class='header'>
-                        <hr>
-                        <h3>TEAM INFORMATION</h3>
-                        <img id="img-side" src="/img/logo.png" />
+                <div class="black-tab-container clearfix">
+                    <div>
+                        <div>
+                            <h1>TEAM INFORMATION</h1>
+                        </div>
```

Comment:
"Team information" is a little too long and sticks out past its bounding box in the tab, making it look off-center. Can you change this to just "Team info" so it looks nicer?